config_frontend/setupTests.js

```diff
@@ -15,4 +15,5 @@ import 'react-testing-library/cleanup-after-each';
 import fetchMock from 'fetch-mock';
 
 fetchMock.catch();
+fetchMock.config.warnOnFallback = false;
```

Comment:
This should be reverted. The reason for the warning is to identify tests where network requests are not being stubbed as they should. We should aim to clean these up rather than ignoring them as they may lead to flaky tests in some environments.

Reply:
We have  reverted this change and both the functionality and tests are still working so we will leave it out.